### PR TITLE
Add draft for java module #100

### PR DESCRIPTION
**Pull request by Thomas-Stewart**
Adding skeleton draft for Java module here. When discussing with @ash-leigh, we thought it beneficial to not break the topics into weeks, but keep them separate. That way the timeline for going through each topic can be more flexible depending on the time needed to go through it.

I'm not sure if this should go in the syllabus or in a separate repo.

Let me know if you have any questions!

### CODE REVIEW

**Comment by Amwam** [comment]
@Thomas-Stewart were trying to move all the modules into topics (all the ones in master have been converted) so definitively a 👍

**Review by kabaros — APPROVED** [review]

**Comment by Thomas-Stewart** [comment]
Any update on this?

**Comment by Thomas-Stewart** [comment]
@kabaros @Amwam

**Comment by Amwam** [comment]
@Thomas-Stewart Is this finished? I can see a lot of empty files

File: SUMMARY.md [hunk 1]
```diff
@@ -19,6 +19,30 @@
   * [Naming patterns](/html-css/naming-patterns.md)
   * [Forms](/html-css/forms.md)
   * [CSS processors](/html-css/css-processors.md)
+* [Java](/java)
```
Comment (Amwam) [inline SUMMARY.md]: This should probably go towards the end, as it isn't a core module, we don't want the students confusing Java vs Javascript right at the beginning of the course

**Comment by Thomas-Stewart** [comment]
@Amwam Must be my mistake. I thought the workflow was to add a skeleton and then be able to make additions to it. So should we be working on my repo then make a pull once it's mostly complete?

**Comment by Amwam** [comment]
@Thomas-Stewart I just wanted to make sure you hadn't forgot to push something, I'm not really sure how you guys are collaborating on this module. If you move the module in the Summary file, i'll merge it in.

**Comment by kabaros** [comment]
hi @Thomas-Stewart, yeah I approved it a while ago. I assumed you would continue with the work on your branch and update the PR. The skeleton looks good and it's exactly the kind of topics we want to cover in an intro module like this.

The comments would come once you add the contents - The only point that I want to raise is just to assume they know JavaScript when they take this module, and try to clarify where Java is different from JS especially at the beginning when talking about types and such.

As for our timeline, I think we'll start doing this end of November/beginning of December as a followup to our main class once it finishes.

**Comment by Thomas-Stewart** [comment]
Thanks guys! I went ahead and moved the module in the summary file.

**Pull request merged**